### PR TITLE
fix mysql

### PR DESCRIPTION
**Pull request by wonderflow**
## Changes
   
- [ ] feature1
- [ ] feature2
- [ ] fixbug1
- [ ] fixbug2
 
## Reviewers

- [ ] @[someone] please review
- [ ] @[someotherone] please review

## Wiki Changes
 
- options1...
- options2...
    
## Checklist
   
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] Wiki updated

### CODE REVIEW

**Comment by redHJ** [comment]
lgtm

File: reader/sql/sql.go [hunk 1]
```diff
@@ -1096,7 +1088,7 @@ func (r *Reader) execCountDB(curDB string, now time.Time, recordTablesDone Table
 
 		// 每张表的记录数
 		var tableSize int64
-		tableSize, err = r.execTableCount(db, idx, curDB, rawSql)
+		tableSize, err = r.execTableCount(connectStr, idx, curDB, rawSql)
```
Comment (redHJ) [inline reader/sql/sql.go]: countdb这边是不是应该也要返回rows.err()?

Comment (redHJ) [reply reader/sql/sql.go]: 并且重试？

Comment (wonderflow) [reply reader/sql/sql.go]: 这个我觉得没有那么重要

Comment (wonderflow) [reply reader/sql/sql.go]: 重试反而重复读数据了

**Pull request merged**